app/src/components/integrations/integrationProviders/jiraIntegration/jiraSettings/jiraSettings.jsx

```diff
@@ -95,7 +95,7 @@ export class JiraSettings extends Component {
     } = this.props;
 
     this.props.showModalAction({
-      id: 'addIntegrationModal',
+      id: 'createProjectIntegrationModal',
```

Comment:
Please recheck modal title for edit modal on the admin page. I don't see any title at least in jira server plugin